packages/dd-trace/src/appsec/iast/overhead-controller.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const OVERHEAD_CONTROLLER_CONTEXT_KEY = 'oce'
+const GLOBAL_CONTEXT_RESET_INTERVAL = 30000
+
+const REPORT_VULNERABILITY = 'REPORT_VULNERABILITY'
+
+let globalContextResetInterval
+const GLOBAL_OCE_CONTEXT = {}
+
+const OPERATIONS = {
+  REPORT_VULNERABILITY: {
+    hasQuota: (context) => {
+      const reserved = context && context.tokens[REPORT_VULNERABILITY] > 0
+      if (reserved) {
+        context.tokens[REPORT_VULNERABILITY]--
+      }
+      return reserved
+    },
+    name: REPORT_VULNERABILITY,
+    initialTokenBucketSize: 2,
+    initContext: function (context) {
+      context.tokens[REPORT_VULNERABILITY] = this.initialTokenBucketSize
+    }
+  }
+}
+
+function _getNewContext () {
+  const oceContext = {
+    tokens: {}
+  }
+
+  for (const operation in OPERATIONS) {
+    OPERATIONS[operation].initContext(oceContext)
+  }
+
+  return oceContext
+}
+
+function _getContext (iastContext) {
+  if (iastContext && iastContext[OVERHEAD_CONTROLLER_CONTEXT_KEY]) {
+    return iastContext[OVERHEAD_CONTROLLER_CONTEXT_KEY]
+  }
+  return GLOBAL_OCE_CONTEXT
+}
+
+function acquireRequest (rootSpan) {
+  return (rootSpan.context().toSpanId() % 3) === 0
```

Comment:
Are you sure that `toSpanId()` returns a number? I've seen that in the implementation:
```javascript
  toSpanId () {
    return this._spanId.toString(10)
  }
```

Reply:
Forget this comment, `'2' % 3 === 2`